EIPS/eip-ueem.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Universal Enshrined Encrypted Mempool
```

Comment:
```suggestion
title: Encrypted Transaction Pool
```
Note: I am not an editor.

Universal isn't used anywhere else in the EIP.
Enshrined is implied as an EIP.
Ethereum has transaction pools.